models/patchArticle.js

```diff
@@ -1,16 +1,17 @@
 const db = require('../db/connection.js');
 
 const patchArticle = (votes, articleId) => {
-    return db.query(`UPDATE articles SET votes = (votes + $1) WHERE article_id = $2 RETURNING *;`, [votes, articleId])
+   
+ 
+  if(!Number.isInteger(votes)){
+    return Promise.reject({status: 400, msg: 'Invalid input'});
+  }
+  
+  
+  return db.query(`UPDATE articles SET votes = (votes + $1) WHERE article_id = $2 RETURNING *;`, [votes, articleId])
     .then(({ rows }) => {
-        const articleInfo = rows[0];
-        if (articleInfo === {}) {
-          return Promise.reject({
-            status: 400,
-            msg: 'Invalid input',
-          });
-        }
-        return articleInfo;
+       
+        return rows[0];
       })
     .catch((err) => {
         console.log(err);
```

Comment:
You don't need a catch block on the model and controller, id remove this and keep your model one as this isn't doing anything anyway